renku/api/datasets.py

```diff
@@ -105,6 +108,14 @@ def store_dataset(self, dataset):
         with dataset.__reference__.open('w') as f:
             yaml.dump(dataset.asjsonld(), f, default_flow_style=False)
 
+    def remove_dataset(self, path):
```

Comment:
how is it linked to a dataset? should it take `Dataset` instance?

Reply:
Why allocate additional object if you don't need it?